src/ui/common/src/components/workflows/SideSheets/OperatorResultsSideSheet.tsx

```diff
@@ -22,9 +22,9 @@ import {
 } from '../../../utils/operators';
 import { ExecState } from '../../../utils/shared';
 import DataPreviewer from '../../DataPreviewer';
+import LogViewer from '../../LogViewer';
```

Comment:
Could you post a screenshot of the OperatorResultsSideSheet with this new log viewer component please? Curious to see how it looks in the side sheet.

Reply:
<img width="1433" alt="image" src="https://user-images.githubusercontent.com/30596854/192061328-13aedb7e-4c36-4a5d-be2c-7eb4eb207fd2.png">